setup.cfg

```diff
@@ -0,0 +1,2 @@
+[flake8]
+max-line-length = 120
```

Comment:
We can keep it at 120 for now but I have a feeling we may have to increase this later when we start doing weird chaining